src/plugins/intel_cpu/src/emitters/README.md

```diff
@@ -0,0 +1,112 @@
+# JIT emitters
+
+## Abstract
+
+Emitter defines an interface for classes repsonsible for code emission. Emitters are designed to implement device specific Instruction Set Architecture (ISA) that might be used as common functions for complex primitives or building blocks for a tensor compiler.
+JIT emitters is a type of emitter designed for just-in-time code generation. JIT emitters architecture involves the following basic principles:
+ * Input and output arguments might be passed via register file, stack or immediates.
+ * Callee (JIT Emitter) is required to preserve registers state which are out of arguments list, caller (e.g. JIT Kernel) is reuired to generate correct emitter call in accordance with its custom ABI.
+ * JIT Emitters are designed as most efficient code generation basic blocks due to their ability to brake default ABI conventions and minimize (or even fully avoid) function call overheads.
+ 
+For just in time source code generation `Xbyak JIT Assembler` is used:
+ * [Xbyak for X64](https://github.com/herumi/xbyak)
+ * [Xbyak for ARM64](https://github.com/fujitsu/xbyak_aarch64)
+
+Emitters are splitted into two groups based on their usage model: common plugin emitters for complex kernels ([plugin emitters](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/src/emitters/plugin)) and basic blocks for tensor compiler ([snippets emitters](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/src/emitters/snippets)).
+
+## Development
+
+Each emitter is linked with OpenVINO operation. For example for `plugin emitters`:
+ * Element-wise `plugin emitters` are linked in element-wise JIT kernel `jit_uni_eltwise_generic::create_eltwise_emitter` method which implementation depends on platform:
+   * [X64 specific](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/eltwise.cpp)
+   * [ARM64 SIMD specific](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/kernels/aarch64/jit_uni_eltwise_generic.cpp)
+
+JIT emitters are inherited from `jit_emitter` base class. The base class implementation depends on architecture:
+ * X64: [jit_emitter.hpp](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/x64/jit_emitter.hpp)
+ * ARM64: [jit_emitter.hpp](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_emitter.hpp)
+
+### Class diagram
+JIT emitters should be inherited from `jit_emitter` base class and it's usage should be added in `jit_uni_eltwise_generic` ARM64 kernel. 
```

Comment:
`jit_uni_eltwise_generic` -> `jit_kernel`

Reply:
updated in this way: `jit_uni_eltwise_generic` -> platform dependent JIT kernel.